Makefile

```
@@ -0,0 +1,24 @@
+SHELL=/bin/bash -euo pipefail
+
+export GO111MODULE        := on
+export PATH               := .bin:${PATH}
+
+.PHONY: install
+install: package-lock.json  django-ory-cloud/poetry.lock 
+		npm i
```

With the current setup, it runs `npm i` every time you run `make test`. That's slow and unnecessary. Here is a setup that runs `npm i` only once in the beginning. Then only if there are changes in the dependencies.

```suggestion
node_modules: package-lock.json  django-ory-cloud/poetry.lock 
	npm i
	touch node_modules
```